abnormal-security/info.json

```diff
@@ -0,0 +1,1259 @@
+        },
+        {
+          "title": "Mock Data",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "checkbox",
+          "name": "mock-data",
+          "tooltip": "Returns test data if set to True.",
+          "description": "Returns test data if set to True.",
+          "value": false
+        }
+      ],
+      "output_schema": {}
+    },
+    {
+      "operation": "get_detection_reports",
+      "annotation": "get_detection_reports",
+      "category": "investigation",
+      "title": "Get Detection Reports",
+      "description": "Get a list of Detection 360° reports that you have submitted and view corresponding details for each case, including report summaries, statuses, message analyses, and more.",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Inquiry Type",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "type": "select",
+          "name": "inquiry_type",
+          "tooltip": "Specify whether to fetch missed attacks or false positives.",
+          "description": "Specify whether to fetch missed attacks or false positives.",
+          "options": [
+            "MISSED_ATTACK",
+            "FALSE_POSITIVE"
+          ]
+        },
+        {
+          "title": "Start",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "text",
+          "name": "start",
+          "tooltip": "The start of the datetime range, as an RFC 3339 timestamp, to fetch Detection 360° cases from. Defaults to 30 days before end.",
+          "description": "The start of the datetime range, as an RFC 3339 timestamp, to fetch Detection 360° cases from. Defaults to 30 days before end.",
+          "placeholder": "eg: 2022-01-01T00:00:00Z"
+        },
+        {
+          "title": "End",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "text",
+          "name": "end",
+          "tooltip": "The end of the datetime range, as an RFC 3339 timestamp, to fetch Detection 360° cases from. Defaults to the current time.",
+          "description": "The end of the datetime range, as an RFC 3339 timestamp, to fetch Detection 360° cases from. Defaults to the current time.",
+          "placeholder": "eg: 2022-01-01T00:00:00Z"
+        },
+        {
+          "title": "Status",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "multiselect",
+          "name": "status",
+          "tooltip": "Only Detection 360° cases with these statuses will be retrieved. Defaults to all statuses.",
+          "description": "Only Detection 360° cases with these statuses will be retrieved. Defaults to all statuses.",
+          "options": [
+            "UNREVIEWED",
```

Comment:
Make the options more friendly looking for UI.

Reply:
done